sycl/test/matrix/matrix-tf32-test.cpp

```diff
@@ -0,0 +1,164 @@
+// RUN: %clangxx -fsycl -O2 -DSYCL_EXT_ONEAPI_MATRIX_VERSION=4 %s -o %t.out
+#include <iostream>
```

Comment:
Shouldn't this kind of runtime test be placed in llvm-test-suite/SYCL/Matrix instead?

Reply:
I will update the header to only check compilation

Reply:
If we use -emit-llvm and FileCheck shouldn't we add checks for the actual IR generated by the compiler?